### ISSUE TITLE
[23.1] How to change aspect in ModelDifference DetailView?

### ISSUE THREAD

**Issue by allshib** [eXpandFramework/eXpand#1034]
When I open the DetailView of a model, it shows "Default Language". I want to change the language of the model in this view to my own. But I didn't see the Language Switcher. How can I change Aspect or set my default Aspect?

**Comment by apobekiaris** [comment]
this worked for me

![image](https://github.com/eXpandFramework/eXpand/assets/159464/3c30860b-1369-4b0f-b808-c173de219278)

**Comment by allshib** [comment]
Thanks for the reply. After changing Preferred Aspect in the view, the model is not updated.

**Comment by apobekiaris** [comment]
worked fine on my side

![image](https://github.com/eXpandFramework/eXpand/assets/159464/4051b8f7-3201-4369-8c61-f05bceae3a87)
![image](https://github.com/eXpandFramework/eXpand/assets/159464/7ea9d2a5-1d1a-4d01-b86c-ceba33eeefc3)


provide exact repro instructions

**Comment by allshib** [comment]
Thanks for the reply. I'll try this.

**Comment by allshib** [comment]
I looked into the problem and this is what I found. What you show does work correctly, but my problem is a bit different. I found a way to get the language button back. I have 2 versions of an application: .NET Framework 4.7.2 and the same application on .NET 6. After reverting the button, the problem on .NET Framework 4.7.2 disappeared, but unfortunately not on .NET 6. I will try to describe my problem in more detail. After returning the button to .NET 6, the model localizations specified in the model localization files were not loaded into it (everything loaded fine in .NET Framework 4.7.2). It is possible that the resources in .NET 6 version are not loaded somewhere.
![image](https://github.com/eXpandFramework/eXpand/assets/98755012/f9e2b2a4-04bb-41a3-b836-138f6d0c35da)
